apiroutes.go

```diff
@@ -803,6 +821,36 @@ func (c *appContext) getBlockRangeSteppedSummary(w http.ResponseWriter, r *http.
 	fmt.Fprintf(w, "]")
 }
 
+func (c *appContext) getBlockSummaryByDate(w http.ResponseWriter, r *http.Request) {
+	date := getDateCtx(r)
+	blocks, er := c.BlockData.GetBlockSummaryByTime(date*86400, (date+1)*86400)
```

Comment:
Might as well keep the int64 all over and keep the units in seconds.

Also, and alt solution to day rounding:  https://play.golang.org/p/9jVLG6vQci